src/views/Settings.vue

```diff
@@ -380,6 +439,56 @@ export default {
     getReadableTime(timestamp) {
       return moment(timestamp).format("MMM D, h:mm:ss a");
     },
+    async changeName() {
+      if (this.newName.length === 0) {
+        return this.$bvToast.toast("Trying to set an empty name...", {
```

Comment:
I would replace this message with "You name must not be empty" or something like that